policyengine_us/parameters/gov/states/ne/tax/income/agi/subtractions/subtractions.yaml

```diff
@@ -0,0 +1,16 @@
+description: Nebraska subtracts these sources from adjusted gross income.
+values:
+  2021-01-01:
+    - ne_social_security_subtraction
+  2024-01-01:
+    - ne_social_security_subtraction
```

Comment:
Right? Per the screenshot in the issue, and the legal text, the threshold no longer applies. Or if we want to stay closer to the law, keep the fraction but make the threshold infinite.

> (14)(a) For taxable years beginning or deemed to begin on or after January 1, 2021, under the Internal Revenue Code of 1986, as amended, federal adjusted gross income shall be reduced by a percentage of the social security benefits that are received and included in federal adjusted gross income. The pertinent percentage shall be:

> 
> (i) Five percent for taxable years beginning or deemed to begin on or after January 1, 2021, and before January 1, 2022, under the Internal Revenue Code of 1986, as amended;
> 
> (ii) Forty percent for taxable years beginning or deemed to begin on or after January 1, 2022, and before January 1, 2023, under the Internal Revenue Code of 1986, as amended;
> 
> (iii) Sixty percent for taxable years beginning or deemed to begin on or after January 1, 2023, and before January 1, 2024, under the Internal Revenue Code of 1986, as amended; and
> 
> (iv) One hundred percent for taxable years beginning or deemed to begin on or after January 1, 2024, under the Internal Revenue Code of 1986, as amended.
> 
> (b) For purposes of this subsection, social security benefits means benefits received under the federal Social Security Act.
> 
> (c) For taxable years beginning or deemed to begin on or after January 1, 2021, and before January 1, 2024, under the Internal Revenue Code of 1986, as amended, a taxpayer may claim the reduction to federal adjusted gross income allowed under this subsection or the reduction to federal adjusted gross income allowed under subsection (13) of this section, whichever provides the greater reduction.

```suggestion
    - taxable_social_security_benefits
```

Reply:
Does the follow not imply 100%:
`One hundred percent for taxable years beginning or deemed to begin on or after January 1, 2024,`

Reply:
Yes but it also removes the threshold. Please add a test for this.